UI/JustAssembly/ViewModels/ComparisonSessionViewModelBase.cs

```diff
@@ -13,8 +13,14 @@ abstract class ComparisonSessionViewModelBase : NotificationObject, IOldToNewTup
 
         public ICommand SwapPathsCommand { get; private set; }
 
-        public ComparisonSessionViewModelBase(string header)
+        public ComparisonSessionViewModelBase(string header, string[] args)
         {
+            if (args != null && args.Length == 2)
```

Comment:
This logic should be only part of the `AssembliesComparisonViewModel` constructor, since `FolderComparisonViewModel` does not support this feature yet.

Reply:
Done. I added an empty constructor to `AssembliesComparisonViewModel` so the `NewSessionViewModel` isn't polluted with the args.